### DISCUSSION TITLE
Start at page n (instead of 1)

### DISCUSSION THREAD

**Discussion by kenNash** [l1xnan/obsidian-better-export-pdf#342] [Q&A]
Hi!

Is it possible?

**Comment by l1xnan** [comment]
It is not easy to implement. In what scenarios is it necessary to start page numbers from n instead of 1?

**Comment by kenNash** [comment]
E.g., in a minute book, where many tens of minutes of assembly deliberations are collected.
For any new minute, it would be veeeery handy not to start the PDF export from scratch.

Thanks for your work and help.
